docs/source/running.rst

```diff
@@ -181,6 +181,24 @@ is simple to run from the command, e.g:
 .. note::
    Uvicorn is as easy to install as ``pip install uvicorn``
 
+
+Django
+^^^^^^
+
+`Django`_ is a Python web framework that encourages rapid development and clean, pragmatic design.  Assuming
+a Django install/enabled application:
+
+.. code-block:: bash
+
+    python3 manage.py runserver
+
+In this section we create a sample Django project and use pygeoapi as an pluggable, embedded application.
```

Comment:
I would rephrase with the common sentence

```text
To integrate pygeoapi as part of another Django project in a pluggable way the truly impatient developers can see `examples/django/sample_project/README.md` for a complete Django application.
As a result, your application will be available at http://localhost:5000/ and pygeoapi will be available at http://localhost:5000/oapi
```

Reply:
Done